dev/deps/spark-deps-hadoop-2.6

```diff
@@ -151,8 +151,8 @@ metrics-jvm/3.1.5//metrics-jvm-3.1.5.jar
 minlog/1.3.0//minlog-1.3.0.jar
 netty-all/4.1.47.Final//netty-all-4.1.47.Final.jar
 netty/3.9.9.Final//netty-3.9.9.Final.jar
```

Comment:
please make sure you run this script to update the classpath files
```
./dev/test-dependencies.sh --replace-manifest
```

Reply:
done